docs/examples/numba_func/Numba_func.ipynb

```diff
@@ -0,0 +1,242 @@
+{
```

Comment:
<blockquote>As you can see we don't specify number of samples in batch. That's because this function is run on a per-sample basis. </blockquote><p>As you can see we don't specify <strong>the </strong>number of samples in <strong>the</strong> batch. That's because this function is run on a per-sample basis. </p><p> </p><blockquote>Note: Please remember passing incorrect outputs and inputs types to <code>run_fn_sig</code> will result in unexpected behaviour. </blockquote><p>Note: Passing incorrect outputs and inputs types to <code>run_fn_sig</code> will result in undefined behaviour. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/2817/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>